Mathlib/LinearAlgebra/Multilinear/Basic.lean

```diff
@@ -653,26 +671,29 @@ end ApplySum
 
 /-- Restrict the codomain of a multilinear map to a submodule.
 
-This is the multilinear version of `linear_map.cod_restrict`. -/
+This is the multilinear version of `LinearMap.codRestrict`. -/
 @[simps]
 def codRestrict (f : MultilinearMap R M₁ M₂) (p : Submodule R M₂) (h : ∀ v, f v ∈ p) :
     MultilinearMap R M₁ p where
   toFun v := ⟨f v, h v⟩
-  map_add' _ v i x y := Subtype.ext <| MultilinearMap.map_add _ _ _ _ _
-  map_smul' _ v i c x := Subtype.ext <| MultilinearMap.map_smul _ _ _ _ _
+  map_add' _ _ _ _ := Subtype.ext <| MultilinearMap.map_add _ _ _ _ _
+  map_smul' _ _ _ _ := Subtype.ext <| MultilinearMap.map_smul _ _ _ _ _
 #align multilinear_map.cod_restrict MultilinearMap.codRestrict
+#align multilinear_map.cod_restrict_apply_coe MultilinearMap.codRestrict_apply_coe
 
 section RestrictScalar
 
+/- Porting note: These used to be a single variable statement, had to split them up to two so that
+the universe metavariable is instantiated before typeclass search runs in `IsScalarTower`. -/
 variable (R) {A : Type _} [Semiring A] [SMul R A] [∀ i : ι, Module A (M₁ i)] [Module A M₂]
-  [∀ i, IsScalarTower R A (M₁ i)] [IsScalarTower R A M₂]
+variable [∀ i, IsScalarTower R A (M₁ i)] [IsScalarTower R A M₂]
```

Comment:
Revisiting this, I wonder if the actual problem is that `(R)` is not a binder update but a new variable.
Suggesting that `variable (R)` should be on its own line.